controllers/horizon_controller.go

```diff
@@ -547,3 +608,32 @@ func (r *HorizonReconciler) ensureHorizonSecret(
 
 	return nil
 }
+
+func (r *HorizonReconciler) renderMemcached(instance *horizonv1alpha1.Horizon) *memcachedv1.Memcached {
```

Comment:
Eventually this and checkMemcachedSvc shuld be part of memcached operator, I think

Reply:
Like a memcachedv1.Create() interface or something you mean?

Or maybe we could add something to lib-common if lots of operators will need to implement this.

Reply:
I'm not too sure about the function name but the same will be needed by the other operators to deploy memcached, so having this logic in a common, reusable space would make sense.